direnvrc

```diff
@@ -197,8 +197,39 @@ _nix_argsum_suffix() {
 }
 
 nix_direnv_watch_file() {
+  # shellcheck disable=2016
+  log_error '`nix_direnv_watch_file` is deprecated - use `watch_file`'
   watch_file "$@"
-  nix_watches+=("$@")
+}
+
+# set by nix build - see ./default.nix
+JQ=
+
+_jq() {
+  if [[ -n $JQ ]]
+  then
+    $JQ "$@"
+  elif has jq
+  then
+    jq "$@"
+  else
+    nix-shell --packages jq --run "$(join_args jq "$@")"
```

Comment:
It makes we wonder if we could just use `nix eval` and parse the json... it probably has better performance than evaluating and downloading `jq`.